frontend/pulse3d/src/components/basicWidgets/UsageProgressWidget.js

```diff
@@ -83,17 +98,53 @@ export default function UsageProgressWidget({ colorOfTextLabel }) {
     }
   }, []);
 
+  const DropDownElement = (
+    <DropDownMenu
+      items={["Upgrade Plan", "Add New Plan"]}
```

Comment:
do both of these items do the same thing? i think clicking the upgrade button should popup the modal, no dropdown is probably needed?